charts/firefly/templates/_helpers.tpl

```diff
@@ -48,6 +48,13 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Create Firefly Node Name
+*/}}
+{{- define "firefly.nodeName" -}}
```

Comment:
This assumes that the `.Release.Namespace` is unique for every installation. But if FF is running in different clusters they might always use the `default` namespace for example (I know of one user that does this).

So just would ask we add a sort of `nodeNameOverride` value somewhere to allow folks to define this their own way if they need. See the `fullname` helpers for how best to do overrides.